broker-daemon/orderbook/index.js

```diff
@@ -337,6 +341,39 @@ class Orderbook {
       sequence
     }
   }
+
+  /**
+   * Updates retry count and calculates next delay period if the
+   * MAX_RETRY_INTERVAL hasn't been surpassed
+   *
+   * @private
+   * @return {void}
+   */
+  _updateExponentialDelayPeriod () {
+    if (this._reachedMaxInterval) {
```

Comment:
this seems like an unnecessary state. Can't you just check if the `_delay` is equivalent to the `MAX_RETRY_INTERVAL`? Or just skip this check entirely and let the max logic do the work for you?